src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -91,19 +103,28 @@ def _bulk_create_with_retry(
             raise last_seen_exception
 
     def bulk_record(
-        self, use_case_id: UseCaseKey, org_strings: Mapping[int, Set[str]]
-    ) -> KeyResults:
-        db_read_keys = KeyCollection(org_strings)
+        self, strings: Mapping[UseCaseID, Mapping[OrgId, Set[str]]]
+    ) -> UseCaseKeyResults:
+        db_read_keys = UseCaseKeyCollection(strings)
 
-        db_read_key_results = KeyResults()
-        db_read_key_results.add_key_results(
+        db_read_key_results = UseCaseKeyResults()
+        db_read_key_results.add_use_case_key_results(
             [
-                KeyResult(org_id=db_obj.organization_id, string=db_obj.string, id=db_obj.id)
-                for db_obj in self._get_db_records(use_case_id, db_read_keys)
+                UseCaseKeyResult(
+                    use_case_id=(
+                        UseCaseID(db_obj.use_case_id)
```

Comment:
This is going to throw an error. If we look at the Django DB models.py, we see that the performance table has the `use_case_id` column. However, the release health table does not, so we can't always do db_obj.use_case_id. And since we haven't done the backfill yet, the performance table's `use_case_id` column is going to say `performance`.